source/message/OP_MSG.rst

```diff
@@ -206,6 +215,40 @@ connection. All other errors during processing will be silently dropped, and
 will not result in the connection being closed.
 
 
+moreToCome On Responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When the ``moreToCome`` flag is set on a response it signals to the recipient
+that the sender will send additional responses on the connection. The recipient
+MUST continue to read responses until it reads a response with the ``moreToCome``
+flag not set, and MUST NOT send any more requests on this connection until
+it reads a response with the ``moreToCome`` flag not set. The client MUST
+either consume all messages with ``moreToCome`` flag set or close the connection.
+
+When the server sends responses with ``moreToCome`` flag set, it behaves as if
+the client sent a ``getMore`` request for the returned cursor for each of the
+responses. Each reply will have a unique ``messageId``, and the ``responseTo``
```

Comment:
Comma after ``messageId`` can be removed, since there are only two parts to the sentence (e.g. A and B).

Reply:
This is a comma joining independent clauses, https://www.grammarly.com/blog/comma-before-and/ under "Comma Before And That Joins Two Independent Clauses".

Reply:
Ah, that's a good point!